src/blog/2023/04/flowforge-1-6-released.md

```diff
@@ -0,0 +1,81 @@
+---
+title: FlowForge v1.6 Now Available
+subtitle: FlowForge Now Supports Multi-Instance Node-RED for Complex Application Development
+description: 
+date: 2023-04-13 
+authors: ["ian-skerrett"]
+image: /blog/2023/03/images/release-150.jpg
+---
+
+The new FlowForge 1.6 adds new support for multi-instance Node-RED within a single application and support for logging from remote devices.
+
+<!--more-->
+
+## FlowForge Applications Can Now Support Multi-Instance Node-RED
+
+FlowForge 1.6 expands the scope of applications to now allow for multiple instances of Node-RED. For complex Node-RED applications, it is common to have different flows interacting with other flows or flows deployed to different target environments. The ability to associate all these different flows with a single application makes it easier for the development, test and deployment of these types of complex applications.
+
+
+
+## Access Node-RED logs from remote devices
+
+FlowForge makes it easy to deploy Node-RED out to remote devices. However, once Node-RED has been deployed to the remote device it is often difficult to troubleshoot or debug. Now with FlowForge 1.6, you can get access to the Node-RED logs from remote devices. This makes it much easier to understand and debug the behavior of a remote device.
+
+
+## Other Improvements
+
+
+Update email address verification  [#813](https://github.com/flowforge/flowforge/issues/813)
+
+Reminder email about trial doesn't include a link to FF Cloud [#1815](https://github.com/flowforge/flowforge/issues/1815)
+
+Sign-up coupons improvement [#1788](https://github.com/flowforge/flowforge/issues/1788)
+
+Deprecate FF_PROJECT_* envvars and add FF_INSTANCE_* equivalents [#1844](https://github.com/flowforge/flowforge/issues/1844)
+
+PostHog events & groups first pass [#1922](https://github.com/flowforge/flowforge/pull/1922)
+
+Add Algolia Search in Docs  [#620](https://github.com/flowforge/website/pull/620) 
```

Comment:
Change to: "Introducing search bar to the docs/handbook page"

Reply:
done